paprika/tests/test_analysis.py

```diff
@@ -84,13 +86,12 @@ def setup_free_energy_calculation():
     # Create window directories
     restraints.restraints.create_window_list([rest1, rest2, rest3])
 
-    # Phase abbreviations
-    phase_dict = {"a": "attach", "p": "pull", "r": "release"}
```

Comment:
What was this dictionary used for before?

Reply:
No clue. I just noticed that PyCharm told me that it wasn't used so I removed it to reduce clutter. Probably a remnant from an earlier version of the test file.